compiler/src/dmd/frontend.h

```diff
@@ -8873,7 +8874,9 @@ struct Id final
     static Identifier* naked;
     static Identifier* thread;
     static Identifier* vector_size;
+    static Identifier* noinline;
     static Identifier* noreturn;
+    static Identifier* _deprecated;
```

Comment:
These header additions don't seem to be related to this PR.

Reply:
It's because I forgot to update frontend.h on a previous PR.